src/gei/Commands/MigrateRepo/MigrateRepoCommandHandler.cs

```diff
@@ -116,7 +116,7 @@ public async Task Handle(MigrateRepoCommandArgs args)
               args.UseGithubStorage
             );
 
-            if (blobCredentialsRequired)
+            if (args.UseGithubStorage && blobCredentialsRequired)
```

Comment:
We need to display this message either if  we use the GH owned storage OR blob creds (`blobCredentialsRequired` is `true`):

```suggestion
            if (args.UseGithubStorage || blobCredentialsRequired)
```